packages/uiweb/src/lib/components/space/SpaceWidget/WidgetHeader.tsx

```diff
@@ -0,0 +1,227 @@
+import React from 'react';
+import styled from 'styled-components';
```

Comment:
```suggestion
import styled from 'styled-components';

```
Add new line after import

Reply:
same for all the files..